_posts/2022-07-20-developer-cli.markdown

```diff
@@ -53,7 +53,7 @@ device1		running		46 seconds ago
 
 Once you have a registered edge device, you can deploy workloads on it:
 ```shell
-$ flotta add workload -d device1
+$ flotta add workload --device device1
```

Comment:
only 1 hyphen?

Reply:
no, it is `-d` or `--device` which is common practice (and also the way the syntax and autocompletion is generate by Cobra library).
see curl (and many other that behave the same way):
```
→ curl --help
Usage: curl [options...] <url>
 -d, --data <data>   HTTP POST data
 -f, --fail          Fail silently (no output at all) on HTTP errors
 -h, --help <category>  Get help for commands
...
```

in our case it is the same:
```
→ flotta add workload -h
Add a new workload

Usage:
  flotta add workload [flags]

Flags:
  -d, --device string   device to run the workload on
  -h, --help            help for workload
  -i, --image string    image of the workload
  -n, --name string     name of the workload to add
```